tools/topology/topology2/platform/intel/sdw-amp-generic.conf

```diff
@@ -175,6 +174,25 @@ IncludeByKey.SDW_AMP_FEEDBACK {
 						copier_type	"ALH"
 						type		"dai_out"
 						node_type $ALH_LINK_INPUT_CLASS
+						#Use 4 channel format if there are 2 amps on different sdw links
+						IncludeByKey.NUM_SDW_AMP_LINKS {
+						"2"	{
+								Object.Base.audio_format [
```

Comment:
@bardliao i'd explicitly use Object.Base.input_audio_format{} and output_audio_format{} in all new additions. I am going to deprecate the audio_format class soon